### DISCUSSION TITLE
Incorrect URL in Github Resources

### DISCUSSION THREAD

**Discussion by shmohit2002** [community/community#72202] [Other Feature Feedback, Questions, & Ideas]
I recently engaged with the learning pathways offered by Github and noticed an incorrect hyperlink on [this page](https://resources.github.com/learn/pathways/automation/essentials/application-testing-with-github-actions). The link in question is located near the “Up next:” text. It currently redirects to [this link](https://resources-hub.githubapp.com/learn/pathways/automation/essentials/configure-your-deployment-environment/?utm_campaign=2024Q2-Learning-Pathways&utm_medium=product&utm_source=github), which appears to be incorrect. Instead, it should point to [this correct page](https://resources.github.com/learn/pathways/automation/essentials/configure-your-deployment-environment/?utm_campaign=2024Q2-Learning-Pathways&utm_medium=product&utm_source=github). Upon clicking the erroneous link, I was redirected to a Github Okta page, which I believe is not the intended behavior.

Furthermore, I observed another instance of this issue on the same page where the domain ‘resources-hub.githubapp.com’ was used instead of the correct ‘resources.github.com’.

I believe these issues are worth addressing to ensure a seamless user experience. Could you please provide a resolution or further insight into this matter? Thank you.

**Comment by klintron** [comment]
Thanks for pointing that out! I just fixed it, and the caches should be updated soon.
